R/pkg/R/client.R

```diff
@@ -61,6 +61,11 @@ generateSparkSubmitArgs <- function(args, sparkHome, jars, sparkSubmitOpts, pack
 }
 
 checkJavaVersion <- function() {
+  if (is_windows()) {
+    # See SPARK-24535
```

Comment:
But this just would end up skipping all tests on Windows ? I'm not sure if thats the right move here. Is there some way to run some tests on Windows ?

Reply:
it would - I guess we need to root cause the reason why checkJavaVersion() fails on Windows/jdk 8 - otherwise sparkR.session() would just fail and no way to continue.

Reply:
updated